packages/schema/src/hooks/resolve.ts

```diff
@@ -170,6 +170,8 @@ export const resolveDispatch =
     })
   }
 
+export const resolveExternal = resolveDispatch
```

Comment:
I think it will be clearer if we rename this to `resolveEventData`.  "External" is ambiguous since response data is also external.

Reply:
oh.  I missed that.